lms/templates/learner_dashboard/programs.html

```diff
@@ -20,9 +20,11 @@
 </%block>
 
 <%block name="pagetitle">${_("Programs")}</%block>
+
 <main id="main" aria-label="Content" tabindex="-1">
-    <div class="program-list-wrapper">
-        <div class="program-cards-container"></div>
-        <div class="sidebar"></div>
-    </div>
+	<div class="program-list-wrapper">
+		<h2 class="sr">${_("Your XSeries Programs")}</h2>
```

Comment:
We should try to avoid mentioning the XSeries brand in Open edX default templates, even if it's screenreader text. Will "Your Programs" work in this context? Otherwise, we could look into using theming.